examples/tauri.rs

```diff
@@ -0,0 +1,26 @@
+#![cfg_attr(
+  all(not(debug_assertions), target_os = "windows"),
+  windows_subsystem = "windows"
+)]
+
+use tauri::Manager;
+use window_vibrancy::{apply_blur, apply_vibrancy, NSVisualEffectMaterial};
+
+fn main() {
+  tauri::Builder::default()
+    .setup(|app| {
+      let win = app.get_window("main").unwrap();
+
+      #[cfg(target_os = "macos")]
+      apply_vibrancy(&win, NSVisualEffectMaterial::AppearanceBased)
+        .expect("Unsupported platform! 'apply_vibrancy' is only supported on macOS");
+
+      #[cfg(target_os = "windows")]
+      apply_blur(&window, Some((18, 18, 18, 125)))
+        .expect("Unsupported platform! 'apply_blur' is only supported on Windows");
+
+      Ok(())
+    })
+    .run(tauri::generate_context!())
+    .expect("error while running tauri application");
+}
```

Comment:
This example won't compile. Instead, you should include a full tauri project i.e one where `tauri.conf.json` and `Cargo.toml` is present.